app/controllers/users_controller.rb

```diff
@@ -0,0 +1,41 @@
+class UsersController < ApplicationController
+
+  def sign_up
+  end
+
+  def sign_up!
```

Comment:
i like that you're trying to hand roll a user auth solution. We need a way to not store any password literals into our database. We want to process passwords into some kind of hashing algorithm. I recommend the `BCrypt` Gem